src/FirebaseProjectManager.php

```diff
@@ -6,6 +6,7 @@
 
 use Illuminate\Contracts\Container\Container;
 use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Support\Arr;
```

Comment:
I know making the manual checks without `Arr` are cumbersome, but I would prefer the introduction of `Arr` in another PR and keep this one focused on the new functionality.